packages/acceptance-tests/pkg-tests-specs/sources/commands/add.test.js

```diff
@@ -81,5 +81,14 @@ describe(`Commands`, () => {
         });
       }),
     );
+
+    test(
+      `it should add node-gyp dependency to yarn.lock if a script uses it`,
+      makeTemporaryEnv({}, async ({path, run, source}) => {
+        await run(`add`, `inject-node-gyp`);
+
+        await expect(readFile(`${path}/yarn.lock`, 'utf8')).resolves.toEqual(expect.stringContaining(`node-gyp: "npm:*"`));
```

Comment:
Didn't know about `expect.stringContaining`! I think ideally the test would be slightly better if you were to parse the lockfile (using `require('@berry/parsers').parseSyml`) and using `.toMatchObject`.